### DISCUSSION TITLE
Make all new windows full screen by default

### DISCUSSION THREAD

**Discussion by jidanni** [i3/i3#4712] [Q&A] [answered]
If I do
```
i3&
a&
b&
c&
```
this will produce [a|b|c]: three windows spread across my screen.

What can I put **in the config** file to make a, b, and c, and any other window,
occupy full screen (be piled on top of each other?)

**Comment by Airblader** [comment]
Only one window can be full-screen at a time.

You might want to look into using stacked or tabbed layout. Alternatively, put them on different workspaces.

**Comment by jidanni** [comment, marked as answer]
(No thanks, one workspace is enough.)
OK, it turns out I want tabbed layout. 
OK, MOD+w gives me that.
But what to put in the configuration file to make it like I pressed MOD+w even on a computer that I haven't used yet?

**Reply by Airblader** [reply]
https://i3wm.org/docs/userguide.html#_layout_mode_for_new_containers
